.github/workflows/tox.yaml

```diff
@@ -0,0 +1,26 @@
+name: tox
+on:
+  pull_request:
+jobs:
+  venv:
+    runs-on: ubuntu-latest
+    permissions: {}
+    timeout-minutes: 20
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+
+      - name: Download toxbox
+        run: podman pull ghcr.io/allisonkarlitskaya/toxbox
```

Comment:
Hmm, that container definition ought to be in cockpit.git containers/, or just be built on the fly. So that it can co-evolve, and not spontaneously break. But it's straightforward enough, and we can change it later if it causes problems.